src/Polly.Core/ExecutionRejectedException.cs

```diff
@@ -8,7 +8,7 @@ namespace Polly;
 /// Exception thrown when a policy rejects execution of a delegate.
 /// <remarks>More specific exceptions which derive from this type, are generally thrown.</remarks>
 /// </summary>
-public abstract class ExecutionRejectedException : Exception
+public class ExecutionRejectedException : Exception
```

Comment:
Just curious, what's the benefit to a non-`abstract` class if there's no way to construct one without deriving from it anyway?

Reply:
That's weird IMHO as having the keyword is more explicit about intent, but I'm not going to die on the hill of not having it vs. suppressing it.

Reply:
Made it abstract again and relaxed the rule. Felt weird to me too.